_posts/2018-10-05-agile-vs-procedures.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Combining rules and procedures with agile culture
+date: 2018-10-05 12:26:35 +0200
+categories: about-us
+---
+
+
+[Last time](/project-roles) I described the process we went through from discovering a problem to creating a solution — [the responsibility matrix](/project-roles). The whole time I had a feeling that we’re not following the agile spirit. It advocates **people and interaction** over **processes and tools**, so a written rule about the roles and responsibilities don’t fit this image.
```

Comment:
"**people and interaction** over **processes and tools**" - Wydaje mi się że wszystko się spina, słowo "over" sugeruje że musimy oddać pierwszeństwo czemuś nad innym. My oddajemy pierwszeństwo decyzjam w zależności od kontekstu i dodajemy ten dokument jako coś drugorzędne, czyli ogólny pattern. Jak by było użyte słowo instead "instead", to wtedy był by problem.

Reply:
Może wyglądać jak ciepianie się do słów, ale prze to że agile jest zdefiniowany 4 zdaniami, to każde słowo wnosi wielką zmiane do interpretacji.

Reply:
yes, I figured that much after finishing writing. I’ll try to rewrite to include that point of view